notebooks/02-ingestion/02-working-with-nested-columns.ipynb

```diff
@@ -63,14 +47,7 @@
     "\n",
     "### Run with Docker\n",
     "\n",
-    "Launch this tutorial and all prerequisites using the `druid-jupyter` profile of the Docker Compose file for Jupyter-based Druid tutorials. For more information, see [Docker for Jupyter Notebook tutorials](https://druid.apache.org/docs/latest/tutorials/tutorial-jupyter-docker.html).\n",
-    "\n",
-    "### Run without Docker\n",
-    "\n",
-    "If you do not use the Docker Compose environment, you need the following:\n",
-    "\n",
-    "* A running Apache Druid instance, with a `DRUID_HOST` local environment variable containing the server name of your Druid router.\n",
-    "* [druidapi](https://github.com/apache/druid/blob/master/examples/quickstart/jupyter-notebooks/druidapi/README.md), a Python client for Apache Druid. Follow the instructions in the Install section of the README file."
+    "Launch this tutorial and all prerequisites using the `druid-jupyter` profile of the Docker Compose file for Jupyter-based Druid tutorials. For more information, see [Docker for Jupyter Notebook tutorials](https://druid.apache.org/docs/latest/tutorials/tutorial-jupyter-docker.html)."
```

Comment:
Not necessarily for this PR, but I think this should link to the README.md for the `learn-druid` repo